src/setuptools_scm/__init__.py

```diff
@@ -52,15 +48,16 @@ def _call_entrypoint_fn(root, config, fn):
         return fn(root, config=config)
     else:
         warnings.warn(
-            "parse functions are required to provide a named argument"
-            " 'config' in the future.",
-            category=PendingDeprecationWarning,
+            f"parse function {fn.__module__}.{fn.__name__}"
+            " are required to provide a named argument"
+            " 'config', setuptools_scm>=8.0 will remove support.",
```

Comment:
Is this 8? Or is this going to be 7 now?

Reply:
its going to be removed in 8.x